projects/plugins/jetpack/modules/masterbar/admin-menu/class-base-admin-menu.php

```diff
@@ -253,6 +254,50 @@ public function is_rtl() {
 		return is_rtl();
 	}
 
+	/**
+	 * Checks for any SVG icons in the menu, and overrides things so that
+	 * we can display the icon in the correct colour for the theme.
+	 */
+	public function override_svg_icons() {
+		global $menu;
+
+		// Only do this if we're not in an API request, as we override the $menu global.
+		if ( $this->is_api_request ) {
+			return;
+		}
+
+		$svg_items = array();
+		foreach ( $menu as $idx => $menu_item ) {
+			if ( count( $menu_item ) > 6 && 0 === strpos( $menu_item[6], 'data:image/svg+xml' ) && 'site-card' !== $menu_item[3] ) {
+				$svg_items[] = array(
+					'icon' => $menu[ $idx ][6],
+					'id'   => $menu[ $idx ][5],
+				);
+				// phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
+				$menu[ $idx ][6] = 'none';
+				// phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
+				$menu[ $idx ][4] .= ' menu-svg-icon';
+			}
+		}
+		if ( count( $svg_items ) > 0 ) {
+			$styles = '.menu-svg-icon .wp-menu-image { background-repeat: no-repeat; background-position: center center } ';
+			foreach ( $svg_items as $svg_item ) {
+				$styles .= sprintf( '#%s .wp-menu-image { background-image: url( "%s" ) }', $svg_item['id'], $svg_item['icon'] );
+			}
+			$styles .= '@supports ( mask-image: none ) or ( -webkit-mask-image: none ) { ';
+			$styles .= '.menu-svg-icon .wp-menu-image { background-image: none; } .menu-svg-icon .wp-menu-image::before { background-color: currentColor; mask-position: center center; mask-repeat: no-repeat; content:"" } ';
+			foreach ( $svg_items as $svg_item ) {
+				$styles .= sprintf( '#%s .wp-menu-image { background-image: none; } #%s .wp-menu-image::before{ mask-image: url( "%s" ) }', $svg_item['id'], $svg_item['id'], $svg_item['icon'] );
+			}
+			$styles .= '}';
+
+			wp_register_style( 'svg-menu-overrides', false, array(), '20210331' );
+			wp_enqueue_style( 'svg-menu-overrides' );
+			wp_add_inline_style( 'svg-menu-overrides', $styles );
+		}
+		l( $svg_items );
```

Comment:
When I applied this branch to Jetpack Beta, I got the following error
```
An error of type E_ERROR was caused in line 298 of the file /srv/htdocs/wp-content/plugins/jetpack-dev/modules/masterbar/admin-menu/class-base-admin-menu.php. Error message: Uncaught Error: Call to undefined function Automattic\Jetpack\Dashboard_Customizations\l() in /srv/htdocs/wp-content/plugins/jetpack-dev/modules/masterbar/admin-menu/class-base-admin-menu.php:298 Stack trace: #0 /wordpress/core/5.7/wp-includes/class-wp-hook.php(292): Automattic\Jetpack\Dashboard_Customizations\Base_Admin_Menu->override_svg_icons('') #1 /wordpress/core/5.7/wp-includes/class-wp-hook.php(316): WP_Hook->apply_filters(NULL, Array) #2 /wordpress/core/5.7/wp-includes/plugin.php(484): WP_Hook->do_action(Array) #3 /wordpress/core/5.7/wp-admin/includes/menu.php(155): do_action('admin_menu', '') #4 /wordpress/core/5.7/wp-admin/menu.php(327): require_once('/wordpress/core...') #5 /wordpress/core/5.7/wp-admin/admin.php(158): require('/wordpress/core...') #6 /wordpress/core/5.7/wp-admin/index.php(10): require_once('/wordpress/core...') #7 {main} thrown
```

Reply:
Oops, that's a debug statement that I thought I'd removed. Thanks, I'll get that fixed.